libs/version/readme.md

```diff
@@ -38,21 +38,44 @@ int main(int argc, char *argv[])
 
 ## 2. Usage in Makefile
 
-At the end of your makefile add the following line:
+At the end of your makefile, you need to include
+```$(BLOCKSDS)/libs/version/blocksds_version.make```.
```

Comment:
You only need single quotes for this and for line 46.

Reply:
:+1: